pkg/apis/sources/register.go

```diff
@@ -32,6 +32,9 @@ const (
 	// SourceDuckLabelValue is the label value to indicate
 	// the CRD is a Source duck type.
 	SourceDuckLabelValue = "true"
+
+	// OIDCInformerSelector is the label selector for the OIDC token creator role and rolebinding informers
+	OIDCInformerSelector = "role=oidc-token-creator"
```

Comment:
nit: can we find something more speaking?
Also when you split it into LabelKey and LabelValue, you can reuse it, when you setup the Role / Rolebinding (`MakeOIDCRole()` and `MakeOIDCRoleBinding()`)

Alternatively we could also only filter on the labelKey (e.g. by using only the selector `oidc`) and not by value. Then we could reuse the same selector later on the filtered service account informers. 
And in `MakeOIDCRole()` we would simply use the label 
```
OIDCSelector: "",
```